### ISSUE TITLE
UI Testing with XLForms

### ISSUE THREAD

**Issue by ninasaveljeva** [xmartlabs/XLForm#749]
Hello,

I'm using XLForms and need to make some UI testing. 
I have a form with several textFields. And I need to set values for all the fields in my tests. How should i do this if XLForm does not set accessibility identificator? Is there any way to set accessibility identificator on form fields? Ive tried to set accessibility label by it is not visible in UI tests.

Thanks. 
Nina

**Comment by mats-claassen** [comment]
Hi Nina, have you tried setting `cellConfig["textField.accessibilityLabel"]`?

**Comment by ninasaveljeva** [comment]
Cool! Thanks! Apparently i've set accessibilityLabel wrong way before. 

Now I've set 
 [row.cellConfig setObject:@"emailLabel" forKey:@"textField.accessibilityLabel"];
for email text field. 
And in uitesting class i can get this field by self.app.textFields[@"emailLabel"]
(I'm using Objective C)

**Comment by kemalbuyukterzi** [comment]
How can i access to accesibilityIdentifier of datepicker in swift related UITest. Or/And How can i change the selected picker value in iutest.
I have tried;
```
app.tables.pickerWheels["Today"].tap()
app.tables.pickerWheels["Fri, 13 May"].tap()
```
or
```
app.tables.cells["sendEventStartDatePicker"].staticTexts["Starts on"].tap()
app.pickerWheels.elementBoundByIndex(1).adjustToPickerWheelValue("Fri, 13 May")
```
This two methods not worked for me.
